src/applications/edu-benefits/1990/helpers.jsx

```diff
@@ -2,6 +2,51 @@ import React from 'react';
 
 import { transformForSubmit } from 'us-forms-system/lib/js/helpers';
 
+export function prefillTransformer(pages, formData, metadata) {
+
+  let newContactInfo = {};
+
+  if (formData && formData.email) {
+    newContactInfo = {
+      email: formData.email,
+      'view:confirmEmail': formData.email
```

Comment:
I wonder if we should make them type this?

Reply:
Good idea- I'll take this part out.